src/interface/src/app/treatments/direct-impacts/metric-filters/metric-filters.component.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { Metric, MetricFiltersComponent } from './metric-filters.component';
+
+export const MockMetrics = [
+  {
+    id: 'ID_1',
+    color: '',
+    label: 'Mock metric 1',
+    active: false,
```

Comment:
I dont think this belongs to the metric itself. The fact that is active should be part of the component `activeMetric: Metric` or something else imo

Reply:
Removed